### PR TITLE
AB#86596 fix of error styling

### PR DESCRIPTION
**Pull request by rafalgajewski**

### CODE REVIEW

File: src/HE.Investments.Common.WWW/HE.Investments.Common.WWW.csproj [hunk 1]
```diff
@@ -19,6 +19,10 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Components\FormField\FormField.cshtml" />
```
Comment (Antek012) [inline src/HE.Investments.Common.WWW/HE.Investments.Common.WWW.csproj:23]: remove this

File: src/HE.Investments.Common.WWW/HE.Investments.Common.WWW.csproj [hunk 2]
```diff
@@ -36,6 +40,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="Components\FormField\FormField.cshtml" />
```
Comment (Antek012) [inline src/HE.Investments.Common.WWW/HE.Investments.Common.WWW.csproj:43]: Move to components section

File: src/HE.Investments.Common.WWW/Components/FormField/FormField.cshtml.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+using He.AspNetCore.Mvc.Gds.Components.Enums;
+using HE.Investments.Common.WWW.Extensions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+
+namespace HE.Investments.Common.WWW.Components.FormField;
+
+public class FormField : ViewComponent
```
Comment (Antek012) [inline src/HE.Investments.Common.WWW/Components/FormField/FormField.cshtml.cs:8]: Where this VC is used?

File: src/HE.Investments.Common.WWW/Components/FormField/FormField.cshtml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+@model (string Title, string Value, string FieldName, ModelExpression ForField, GdsInputPrefixText? Prefix, bool IsInvalid, string Error)
```
Comment (Antek012) [inline src/HE.Investments.Common.WWW/Components/FormField/FormField.cshtml:1]: Does ModelExpression work?

File: src/HE.Investments.Common.WWW/HE.Investments.Common.WWW.csproj (hunk 1 above)
Comment (rafalgajewski) [reply src/HE.Investments.Common.WWW/HE.Investments.Common.WWW.csproj]: Removed.

File: src/HE.Investments.Common.WWW/HE.Investments.Common.WWW.csproj (hunk 2 above)
Comment (rafalgajewski) [reply src/HE.Investments.Common.WWW/HE.Investments.Common.WWW.csproj]: Removed.

File: src/HE.Investments.Common.WWW/Components/FormField/FormField.cshtml (hunk 4 above)
Comment (rafalgajewski) [reply src/HE.Investments.Common.WWW/Components/FormField/FormField.cshtml]: Removed.

**Review by jjankowski87 — APPROVED** [review]

**Review by Antek012 — APPROVED** [review]

**Pull request merged**